examples/README.md

```diff
@@ -3,6 +3,7 @@
 The repository includes the following example scripts:
 - fingerprint
 - gnn
+- acnn
```

Comment:
Do we need to add "Interaction Fingerprint"?

Reply:
Added